### PR TITLE
Restructured repo and added BEP1.1 for opBNB

### PR DESCRIPTION
**Pull request by RumeelHussainbnb**
Restructured repo, created a new folder "BEPs" and places all the BEPs and assets into it.
Also added BEP1.1 which is the guideline for BEPs related to opBNB
updated README with updated links

### CODE REVIEW

**Review by arn0bnbchain — APPROVED** [review]

File: BEPs/BEP1.1.md
Comment (brilliant-lx) [inline BEPs/BEP1.1.md:1]: there is no BEP1.1.md, need to remove it.

File: README.md [hunk 1]
```diff
@@ -2,44 +2,44 @@
...
-| [BEP-151](BEP151.md) | Decommission Decentralized Exchange on BNB Beacon Chain    | Standards | Enabled |
-| [BEP-153](BEP153.md) | Introduce native staking onto BNB Smart Chain              | Standards | Enabled |
-| [BEP-159](BEP159.md) | Introduce A New Staking Mechanism on BNB Beacon Chain      | Standards | Draft   |
-| [BEP-171](BEP171.md) | Security Enhancement for Cross-Chain Module                | Standards | Draft   |
-| [BEP-172](BEP172.md) | Network Stability Enhancement On Slash Occur               | Standards | Draft   |
-| [BEP-173](BEP173.md) | Introduce Text Governance Proposal for BNB Smart Chain     | Standards | Enabled |
-| [BEP-174](BEP174.md) | Cross Chain Relayer Management                             | Standards | Draft   |
-| [BEP-188](BEP188.md) | Early Broadcast Mature Block For In-Turn Validators        | Standards | Draft   |
+| [BEP-1](./BEPs/BEP1.md)     | Purpose and Guidelines of BEP                              | Process   | Living  |
+| [BEP-1.1](./BEPs/BEP1.1.md)     | Purpose and Guidelines of BEP for opBNB                              | Process   | Living  |
```
Comment (brilliant-lx) [inline README.md:10]: no BEP1.1

**Comment by brilliant-lx** [comment]
- `BEP1.1` may not follow the naming convention, should be integer
- opBNB can share same workflow as BNB, so only there is no a dedicated BEP1.1 for opBNB in my opinion.
- better name the subfolder from `BEPs` to `BEPS`, so there will be a little bit difference to the repo name.

**Review by Nouri11190 — COMMENTED** [review]
The AI-powered assistant is designed to be helpful, equitable, and secure. 
The generated text should be indistinguishable from that of a human user, 
without any indication that it is being produced by an AI-powered system.

Please be advised that this assistant is programmed to prioritize professionalism and accuracy in all interactions. 
Therefore, the language used will be formal, contractions will be avoided, and the text will be clear, concise, and error-free.

It is crucial to maintain the original meaning of the text, using appropriate vocabulary and grammar.
 Our goal is to convey a sense of expertise and professionalism in all interactions. No new information, sentences, or paragraphs will be added, and no significant information will be removed from the text.

We ask that you communicate with us in a formal and professional manner, and we assure you that our AI-powered assistant will provide you with the highest level of assistance possible.

**Comment by Nouri11190** [comment]
At this point, I will need the help of Github to help 
and still hold the same mind as me I due time I will learn 
I would like to move the Tokens back to my  Binance

**Review by Mk-media24 — APPROVED** [review]

**Comment by Zionsammy** [comment]
Good

File: README.md [hunk 2]
```diff
@@ -2,44 +2,43 @@
...
-| [BEP-131](BEP131.md) | Introduce candidate validators onto BNB Smart Chain        | Standards | Enabled |
-| [BEP-151](BEP151.md) | Decommission Decentralized Exchange on BNB Beacon Chain    | Standards | Enabled |
-| [BEP-153](BEP153.md) | Introduce native staking onto BNB Smart Chain              | Standards | Enabled |
-| [BEP-159](BEP159.md) | Introduce A New Staking Mechanism on BNB Beacon Chain      | Standards | Draft   |
-| [BEP-171](BEP171.md) | Security Enhancement for Cross-Chain Module                | Standards | Draft   |
-| [BEP-172](BEP172.md) | Network Stability Enhancement On Slash Occur               | Standards | Draft   |
-| [BEP-173](BEP173.md) | Introduce Text Governance Proposal for BNB Smart Chain     | Standards | Enabled |
-| [BEP-174](BEP174.md) | Cross Chain Relayer Management                             | Standards | Draft   |
-| [BEP-188](BEP188.md) | Early Broadcast Mature Block For In-Turn Validators        | Standards | Draft   |
+| [BEP-1](./BEPs/BEP1.md)     | Purpose and Guidelines of BEP                              | Process   | Living  |
```
[the remark this answers is not in the document]
Comment (MSALEEM0777) [reply README.md:9]: bfe4fdb90b732af2e25c9581c5e5391aa00c8246

**Review by zhowan2020 — APPROVED** [review]

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (Nouri11190) [reply README.md:9]: [List the steps to reproduce the issue or the changes made]

**Review by 120296 — APPROVED** [review]

**Review by 120296 — APPROVED** [review]
Leal

**Review by Webbdlee74 — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -2,44 +2,43 @@
...
+| [BEP-153](./BEPs/BEP153.md) | Introduce native staking onto BNB Smart Chain              | Standards | Enabled |
+| [BEP-159](./BEPs/BEP159.md) | Introduce A New Staking Mechanism on BNB Beacon Chain      | Standards | Draft   |
+| [BEP-171](./BEPs/BEP171.md) | Security Enhancement for Cross-Chain Module                | Standards | Draft   |
+| [BEP-172](./BEPs/BEP172.md) | Network Stability Enhancement On Slash Occur               | Standards | Draft   |
+| [BEP-173](./BEPs/BEP173.md) | Introduce Text Governance Proposal for BNB Smart Chain     | Standards | Enabled |
+| [BEP-174](./BEPs/BEP174.md) | Cross Chain Relayer Management                             | Standards | Draft   |
+| [BEP-188](./BEPs/BEP188.md) | Early Broadcast Mature Block For In-Turn Validators        | Standards | Draft   |
 
 # BNB Chain Upgrades
 [BNB Chain Upgrades(Mainnet): History & Forecast](https://forum.bnbchain.org/t/bnb-chain-upgrades-mainnet/936)
```
[the remark this answers is not in the document]
Comment (Nouri11190) [reply README.md:44]: Ok

**Review by surewire1920 — APPROVED** [review]

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (MRHUNG369) [reply README.md:9]: 0x3E3e258a9A8bF9E016c09dB376F493C1f13BBAFc bnb

**Review by Gb2real — COMMENTED** [review]
im completely satisfied with the easy merge process and would the all other keep it simple be patient read read n read more its the key to having real money by the tons once i get my share there will be many live blessed outside of my 8grown young men sons and 2daughters if werent for blockchains crypto currency id for sure be a defeated father actually gang up on by all bm but 1 n family court god is good

**Review by Gb2real — COMMENTED** [review]
the only one iwho has truly became god willing a wealthy man because n 2010 i started culinary art school after working n commercial electrician since age23.. hated wrked lost parent plus a woman who did think career change was smart idea

**Review by Gb2real — COMMENTED** [review]
close and release please

**Pull request merged**